### PR TITLE
Add page "Comparison with other EDA tools"

### LINKED ISSUE #15: Why LibrePCB?
Maybe link to FOSDEM Talk / Slides.

### PR DESCRIPTION
**Pull request by ubruhin**
It's not beautiful, but probably better than nothing. What do you think @dbrgn?

Fixes #15 (more or less)

### CODE REVIEW

File: content/compare/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+- Easy-to-use library manager to install and update libraries - no
+  knowledge about Git and GitHub needed
+- LibrePCB does not bundle (possibly outdated or low-quality) libraries together
+  with the application - you have the full control over what libraries are used
+- Only one library file format for all kinds of library elements - a library can
+  contain symbols, footprints, components and in future also 3D models
+- No broken references - you can rename symbols, footprints, pins, pads etc.
+  without breaking other elements
+- LibrePCB organizes parts by category, thus it's easier and more intuitive to
```
Comment (dbrgn) [inline content/compare/index.md]:
```suggestion
- LibrePCB organizes parts by hierarchical categories, thus it's easier and more intuitive to
```

File: content/compare/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  find the part you're looking for
+- No broken projects caused by updated or removed libraries - LibrePCB projects
+  are self-contained and completely independent of system libraries
+
+**Version control of LibrePCB files makes more fun:**
```
Comment (dbrgn) [inline content/compare/index.md]:
```suggestion
**Version control of LibrePCB files is more fun:**
```

I don't think "make fun" is proper English :)

File: content/compare/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+- All files of LibrePCB projects and libraries are highly optimized for usage of
```
Comment (dbrgn) [inline content/compare/index.md]:
```suggestion
- All files of LibrePCB projects and libraries are highly optimized for
```

File: content/compare/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  version control systems
+- User-related settings are strictly separated from project-related settings so
+  you don't commit annoying, useless changes
```
Comment (dbrgn) [inline content/compare/index.md]:
```suggestion
  you don't commit annoying, useless changes (like your zoom level or canvas position)
```

File: content/compare/index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+- LibrePCB files are canonical - if you save a project without making relevant
```
Comment (dbrgn) [inline content/compare/index.md]:
```suggestion
- LibrePCB files have a canonical file format - if you save a project without making relevant
```

File: content/compare/index.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+**BUT:**
+
+Although LibrePCB has many cool advantages, KiCad is (currently) much more
+powerful. It has an amazing amount of features which allow to design very
+complex PCBs. LibrePCB is still a very young software and thus lacks many
+features needed to design complex PCBs.
+
+So, if you are looking for an intuitive EDA tool to quickly design a simple PCB,
+you should give LibrePCB a try. But if you want to design complex PCBs, LibrePCB
+is probably not the tool you are looking for.
```
Comment (dbrgn) [inline content/compare/index.md]:
```suggestion
is probably not (yet) the tool you are looking for.
```

**Comment by dbrgn** [comment]
Oh, and you should replace minus signs (-) with en dashes (–) if you use them as separators.

File: content/compare/index.md (hunk 5 above)
Comment (ubruhin) [reply content/compare/index.md]: I reworded it to "LibrePCB has a canonical file format".

**Comment by ubruhin** [comment]
> Oh, and you should replace minus signs (-) with en dashes (–) if you use them as separators.

Done.

**Pull request merged**